wizard/WizardModeSelection.qml

```diff
@@ -91,6 +91,7 @@ Rectangle {
 
                 onMenuClicked: {
                     if(appWindow.persistentSettings.nettype == 0){
+                        appWindow.persistentSettings.pruneBlockchain = true;
```

Comment:
?

Reply:
Prune by default for mainnet simple mode

Reply:
Simple mode does not sync a chain so it does not make sense to prune here.

Reply:
Ah you’re right

Reply:
So only necessary for simple mode (bootstrap)